packages/adapter-utils/src/element_source.ts

```diff
@@ -68,8 +68,10 @@ export const buildElementsSourceFromElements = (
   }
 
   const get = async (id: ElemID): Promise<Value> => {
-    const element = elementsMap[id.getFullName()]
-    if (element !== undefined) {
+    const { parent } = id.createTopLevelParentID()
+    const topLevelElement = elementsMap[parent.getFullName()]
+    const element = elementsMap[id.getFullName()] ?? (topLevelElement && resolvePath(topLevelElement, id))
+    if (element) {
```

Comment:
element can be falsy - 0, '', false

```suggestion
    if (element !== undefined) {
```